external/loaders/loaders/batches/save.py

```diff
@@ -39,31 +40,50 @@ def get_parser() -> argparse.ArgumentParser:
             "passed to BatchesLoader.load_batches"
         ),
     )
+    parser.add_argument(
+        "--num-workers",
+        type=int,
+        default=8,
+        help="number of workers to use when saving batches",
+    )
     return parser
 
 
-def main(data_config: str, output_path: str, variable_names: Sequence[str]):
+def main(
+    data_config: str,
+    output_path: str,
+    variable_names: Sequence[str],
+    num_workers: int = 1,
```

Comment:
```suggestion
    num_workers: int = 8,
```

to be consistent with the default num-workers

Reply:
I left it as `num_workers=1` since the argument parser defines the default for external calls to this, and I didn't want the tests attached to `main` to use multiprocessing